data/korath/korath ships.txt

```diff
@@ -309,89 +309,842 @@ ship "Korath Chaser"
+	turret -186 -285.5 "Korath Banisher"
+	turret 186 -285.5 "Korath Warder"
+	turret -131.5 -173.5 "Langrage Hyper-Heaver"
+	turret 131.5 -173.5 "Korath Inferno"
+	turret -131.5 -156 "Korath Inferno"
+	turret 131.5 -156 "Korath Warder"
+	turret 46 -12 "Korath Warder"
+	turret -16.5 16 "Korath Banisher"
+	turret 62 128 "Langrage Hyper-Heaver"
+	turret -75.5 162.5 "Firestorm Turret"
+	turret -204.5 200 "Korath Banisher"
+	turret 204.5 200 "Korath Warder"
+	turret -192.5 257.5 "Korath Warder"
+	turret 192.5 257.5 "Korath Banisher"
+	turret -72.5 338.5 "Langrage Hyper-Heaver"
+	turret 72.5 338.5 "Korath Inferno"
+	turret -95.5 365.5 "Korath Inferno"
+	turret 95.5 365.5 "Korath Banisher"
+	gun -170 -104 "Firelight Missile Bank"
+		angle 315
+	gun 170 -104 "Firelight Missile Bank"
+		angle 45
+	gun -178 -84 "Firelight Missile Bank"
+		angle 290
+	gun 178 -84 "Firelight Missile Bank"
+		angle 70
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Korath Havens are massive nomadic habitats that hold much of their remaining population.  True flying cities, they could be self-contained miniature worlds were it not for their heavy defensive armament and reliance on unsustainable materials, methods, and sources of power."
+	description "  Commission of new Havens is an intensely resource-demanding, labour-intensive, painstaking process fraught with political intrigue. Navarchs fight with the civilian government for resources and yard time they would rather devote to building raiding and defensive fleets, civilians vie with each other for a place on the next ship, and planning committees argue with engineers over what methods and technologies to use. This leads to construction delays, corner-cutting, and over-militarisation of the design at best, and violent feuds at worst."
```

Comment:
I don't recall the term Navarch being used anywhere in the game besides here. Will it be made clear what a Navarch is to the player in future content?

Reply:
Yes, the lore we've been developing is that the Exiles - while having a *de jure* civilian government - are in many areas dominated rather by a military junta of "Navarchs," that is, admiral-rulers.  We do plan to expand upon this in future missions.